jsreport-linux-appservice-docker/azuredeploy.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "appName": {
+      "defaultValue": "[concat('jsreportsapp',uniqueString(resourceGroup().id))]",
+      "type": "string",
+      "metadata": {
+        "description": "Name of the WebApp Running jsreport"
+      }
+    },
+    "storageaccountsku": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ],
+      "metadata": {
+        "description": "Describes the Storage Account Type"
+      }
+    }
+  },
+  "variables": {
+    "servicePlanName": "[concat('serviceplan',uniqueString(resourceGroup().id))]",
+    "StorageName": "[concat('jsreports', uniqueString(resourceGroup().id))]",
+    "storageAccountApiVersion": "2015-06-15",
+    "storageId": "[resourceId('Microsoft.Storage/storageAccounts', variables('StorageName'))]",
+    "storageConnectionStringPrefix":
```

Comment:
use reference to get the uri - reference is not allowed in the vars section so you'll need to move this inline...

Reply:
remove this